docs/css/style.css

```diff
@@ -3,7 +3,10 @@
 .md-footer-nav,
 .md-footer-meta,
 .md-nav__source {
-    background-color: #091440
+    background-color: #3f51b5;
```

Comment:
I'd suggest going with a color from the decred.org/brand guidelines. 

`#1B41B3`, `#2970FF`, or `#4A92FF` look pretty close to what you have used in this PR.

Reply:
Yeah tried the colors from the guidelines, but they all looked too dark or too light (blue I used is the mkdocs template default). The middle blue `#2970FF` the best though if we want to stick to guidelines (which looks fine). Have switched to that.